src/connectors/CronConnector.ts

```diff
@@ -1,63 +1,49 @@
 import { BaseConnector, EventConfiguration } from 'reshuffle-base-connector'
-import Timer = NodeJS.Timer
+import cron from 'node-cron'
 import Reshuffle from '../Reshuffle'
 
-const DEFAULT_EVENT_OPTIONS = { interval: 5000 }
-
 export interface CronEventOptions {
-  interval: number
+  expression: string
+  timezone?: cron.ScheduleOptions['timezone']
 }
 
 export default class CronConnector extends BaseConnector<null, CronEventOptions> {
-  intervalsByEventId: { [eventId: string]: Timer }
+  tasksByEventId: { [eventId: string]: cron.ScheduledTask }
 
   constructor(app: Reshuffle, id?: string) {
-    super(app, undefined, id)
-    this.intervalsByEventId = {}
+    super(app, null, id)
+    this.tasksByEventId = {}
   }
 
-  on(
-    options: CronEventOptions = DEFAULT_EVENT_OPTIONS,
-    handler: any,
-    eventId?: string,
-  ): EventConfiguration {
+  on(options: CronEventOptions, handler: any, eventId?: string): EventConfiguration {
     if (!eventId) {
-      eventId = `CRON/${options.interval}/${this.id}`
+      eventId = `CRON/${this.id}`
```

Comment:
This sets up for not creating a unique id. I would urge to add a `nanoid()` or something

Reply:
Yes true, will add that